### PR TITLE
Sync changes with master

### PR DESCRIPTION
**Pull request by sungam3r**

### CODE REVIEW

File: .github/workflows/build-artifacts-code.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ name: Build artifacts
 on:
   push:
     branches:
+      - master
```
Comment (sungam3r) [inline .github/workflows/build-artifacts-code.yml:8]: @Shane32 It is right?

File: .github/workflows/pr-tests-code.yml [hunk 2]
```diff
@@ -3,20 +3,21 @@ name: Run code tests
 on:
   pull_request:
     branches: 
+      - master
```
Comment (sungam3r) [inline .github/workflows/pr-tests-code.yml:6]: @Shane32 It is right?

File: .github/workflows/pr-tests-docs.yml [hunk 3]
```diff
@@ -3,6 +3,7 @@ name: Run documentation tests
 on:
   pull_request:
     branches: 
+      - master
```
Comment (sungam3r) [inline .github/workflows/pr-tests-docs.yml:6]: @Shane32 It is right?

**Comment by sungam3r** [comment]
`
Testhost process exited with error: It was not possible to find any compatible framework version
The framework 'Microsoft.NETCore.App', version '2.1.0' was not found.
`

Should we keep 2.1 target or just remove it?

**Comment by Shane32** [comment]
Yes it should have `- master` in the workflows.

Yes we should keep the 2.1 target.

I've made the necessary changes

**Review by Shane32 — APPROVED** [review]

**Comment by Shane32** [comment]
I re-did the merge to eliminate the extra commits in the history.

**Pull request merged**